JSON_Bourne/get_webpage.py

```diff
@@ -0,0 +1,112 @@
+from lxml import html
+import requests
+import json
+
+
+def get_info(url):
+    page = requests.get(url)
+    tree = html.fromstring(page.content)
+
+    titles = tree.xpath("//tr/th/a")
+    titles = [t.text for t in titles]
+
+    status = tree.xpath("//table[2]/tbody/tr/td[1]")
+    # status = [s.text for s in status]
+    status_text = list()
+    for s in status:
+        if s.text is None:
+            # This means there is an extra <font> in the XML to change the colour of the value
+            # This only happens when the PV is "Disconnected"
+            # This assumption speeds up the program significantly
+            status_text.append("Disconnected")
+        else:
+            status_text.append(s.text)
+
+    info = tree.xpath("//table[2]/tbody/tr/td[3]")
+    values = list()
+    alarms = list()
+
+    for i in range(len(titles)):
+        if "STARTTIME" in titles[i]:
+            # Time is delimited by \t
+            if info[i].text == "null":
+                values.append("null")
+                alarms.append("null")
+            else:
+                single = info[i].text.split("\t", 2)
+                values.append(single[1])
+                alarms.append(single[2])
```

Comment:
Magic numbers as array indices can make maintenance more difficult and the code more fragile. Use "static" variables if absolutely necessary. Further clarity can sometimes be added by setting up short-lived dummy variables with names that describe what the element is.